python/lsst/ts/observatory/control/base_calsys.py

```diff
@@ -134,7 +137,7 @@ async def perform_flat(self, exptime_dict: dict) -> None:
             If there is some error in the data taking of the camera,
             electrometer or fiber spectrographs
         """
-        raise NotImplementedError()
+        await self.expose_calsys(exptime_dict)
```

Comment:
If this method is just calling `expose_calsys` than we should get rid of one of them. There is no reason to have a method that just call another method unless you are providing backward compatibility or something like that.

Reply:
removed expose_calsys